sources/packages/backend/apps/api/src/services/education-program/education-program.service.models.ts

```diff
@@ -1,14 +1,19 @@
 import { ProgramIntensity, ProgramStatus } from "@sims/sims-db";
+import { Allow } from "class-validator";
 
 export interface ProgramDeliveryTypes {
   deliveredOnSite: boolean;
   deliveredOnline: boolean;
 }
 
-export interface EntranceRequirements {
+export class EntranceRequirements {
```

Comment:
May be or may not be part of the PR. EntranceRequirements is a service model which by mistake we are using in the DTO level in EducationProgramAPIOutDTO. We can create a class for EntranceRequirements  at DTO level and add allow decorator to it.

Reply:
Changes reverted.